tools/src/CSVFileImport.cc

```diff
@@ -280,15 +280,19 @@ void usage() {
             << "                  [-s <size>] [--stripe=<size>]\n"
             << "                  [-c <size>] [--block=<size>]\n"
             << "                  [-b <size>] [--batch=<size>]\n"
+            << "                  [-t <string>] [--timezone=<string>]\n"
             << "                  <schema> <input> <output>\n"
             << "Import CSV file into an Orc file using the specified schema.\n"
+            << "The timezone can be viewed in the directory /usr/share/zoneinfo\n"
```

Comment:
- It should be stated clearly that the timezone is writer timezone of timestamp types.
- You have specified it as a **required_argument** which conflicts with the comment here.